packages/core/src/tools/error.spec.ts

```diff
@@ -71,3 +71,23 @@ describe('formatUnknownError', () => {
     expect(formatted.message).toEqual('Uncaught {"foo":"bar"}')
   })
 })
+
+describe('createHandlingStackTrace', () => {
```

Comment:
Could we have a test with an actual handling stack expectation?

Reply:
What do you mean, by actual handling stack? Are these tests more what you're looking for?
https://github.com/DataDog/browser-sdk/blob/c7d3145240054d9dd115c205912e84c3f59f1339/packages/rum-core/src/boot/rumPublicApi.spec.ts#L283-L290


https://github.com/DataDog/browser-sdk/blob/c7d3145240054d9dd115c205912e84c3f59f1339/packages/core/src/domain/automaticErrorCollection.spec.ts#L51-L59

Reply:
I'd say something like:
https://github.com/DataDog/browser-sdk/blob/fb7e9a1e4549b1d2959413733a22c45cc2ecb3e8/packages/core/src/tools/error.spec.ts#L40-L43
but for handling stack

Reply:
The thing is, the handling stack includes karma files path and calls, that's why I used contains. I can try to use a regex for a more complete ​check if you think it worth it?